packages/multichain-api-middleware/CHANGELOG.md

```diff
@@ -7,8 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.3.0]
+
+### Added
+
+- Add more chain-agnostic-permission utility functions from sip-26 usage ([#5609](https://github.com/MetaMask/core/pull/5609))
+
 ### Changed
 
+- feat: Add more chain-agnostic-permission utility functions from sip-26 usage ([#5609](https://github.com/MetaMask/core/pull/5609))
```

Comment:
Nit: can we remove this entry from here now ?

Reply:
🙏 fixed here: https://github.com/MetaMask/core/pull/5818/commits/2abe4bcad3b8783b0bdeff0d9078e3e94149b6da